src/raygui.h

```diff
@@ -2896,68 +2896,72 @@ RAYGUIDEF void GuiLoadStyle(const char *fileName)
 
-                        Font font = { 0 };
+                            Font font = { 0 };
 
-                        if (charmapFileName[0] != '0')
-                        {
-                            // Load characters from charmap file,
-                            // expected '\n' separated list of integer values
-                            char *charValues = LoadText(charmapFileName);
-                            if (charValues != NULL)
+                            if (charmapFileName[0] != '0')
                             {
-                                int charsCount = 0;
-                                const char **chars = TextSplit(charValues, '\n', &charsCount);   // WARNING: TextSplit only supports 64 strings!
+                                // Load characters from charmap file,
+                                // expected '\n' separated list of integer values
+                                char *charValues = LoadText(charmapFileName);
+                                if (charValues != NULL)
+                                {
+                                    int charsCount = 0;
+                                    const char **chars = TextSplit(charValues, '\n', &charsCount);   // WARNING: TextSplit only supports 64 strings!
 
-                                int *values = (int *)malloc(charsCount*sizeof(int));
-                                for (int i = 0; i < charsCount; i++) values[i] = atoi(chars[i]);
+                                    int *values = (int *)malloc(charsCount*sizeof(int));
+                                    for (int i = 0; i < charsCount; i++) values[i] = atoi(chars[i]);
 
-                                font = LoadFontEx(FormatText("%s/%s", GetDirectoryPath(fileName), fontFileName), fontSize, values, charsCount);
+                                    font = LoadFontEx(FormatText("%s/%s", GetDirectoryPath(fileName), fontFileName), fontSize, values, charsCount);
 
-                                free(values);
+                                    free(values);
+                                }
                             }
-                        }
-                        else font = LoadFontEx(FormatText("%s/%s", GetDirectoryPath(fileName), fontFileName), fontSize, NULL, 0);
+                            else font = LoadFontEx(FormatText("%s/%s", GetDirectoryPath(fileName), fontFileName), fontSize, NULL, 0);
 
-                        if ((font.texture.id > 0) && (font.charsCount > 0)) GuiSetFont(font);
+                            if ((font.texture.id > 0) && (font.charsCount > 0)) GuiSetFont(font);
 
-                    } break;
-                    default: break;
-                }
+                        } break;
+                        default: break;
+                    }
 
-                fgets(buffer, 256, rgsFile);
+                    if ( fgets(buffer, 256, rgsFile) == NULL )
+                    {
+                        // error
```

Comment:
I wasn't sure what to do here, how do we propagate errors to the user in this case?